src/main/java/io/confluent/idesidecar/restapi/clients/SchemaErrors.java

```diff
@@ -25,23 +46,27 @@ public record Error(String message) {
 
   }
 
-  private static final Duration SCHEMA_FETCH_ERROR_TTL = Duration.ofSeconds(30);
+  @ConfigProperty(name = "ide-sidecar.schema-fetch-error-ttl")
+  long schemaFetchErrorTtl;
 
   public static final Map<ConnectionId, Cache<SchemaId, SchemaErrors.Error>> cacheOfCaches = new ConcurrentHashMap<>();
 
-  public static Cache<SchemaId, Error> getSubCache(ConnectionId key) {
+ // getSubCache(ConnectionId key): Retrieves or creates a sub-cache for a specific connection.
+  public Cache<SchemaId, Error> getSubCache(ConnectionId key) {
     return cacheOfCaches.computeIfAbsent(
         key,
-        k -> Caffeine.newBuilder().expireAfterAccess(SCHEMA_FETCH_ERROR_TTL).build());
+        k -> Caffeine.newBuilder().expireAfterAccess(Duration.ofSeconds(schemaFetchErrorTtl)).build());
   }
 
+  // readSchemaIdByConnectionId(ConnectionId connectionId, SchemaId schemaId): Reads a schema error from the cache.
```

Comment:
We typically try to use [Javadocs](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html) style comments to document functionality. The comment used here `//` is a [single-line (or end-of-line) comment](https://www.oracle.com/java/technologies/javase/codeconventions-comments.html) and we reserve those for clarifying explanations embedded in code implementations.

```
/**
* Reads a schema error from the cache.
*/
```

Reply:
thanks! pushed a change